src/main/java/com/gregtechceu/gtceu/utils/fakelevel/Structure.java

```diff
@@ -0,0 +1,77 @@
+package com.gregtechceu.gtceu.utils.fakelevel;
+
+import it.unimi.dsi.fastutil.chars.Char2ObjectMap;
+import it.unimi.dsi.fastutil.chars.Char2ObjectOpenHashMap;
+import it.unimi.dsi.fastutil.chars.CharArrayList;
+import it.unimi.dsi.fastutil.chars.CharList;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import net.minecraft.core.BlockPos;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Structure {
```

Comment:
I think you can remove this. Idk why it still exists, ArraySchema does what this was trying to do